Ops handbook tools (#832)
publiccodenet/about: activities/tool-management/nextcloud.md

## Privacy and GDPR

> We run a self-hosted instance of Nextcloud on a server managed by the trusted [service provider Maadix in Spain.](https://maadix.net/en/terms-conditions)

Reviewer: We implicitly trust all of our tools and providers, or else we wouldn't use them.

```suggestion
We run a self-hosted instance of Nextcloud on a server managed by our [service provider Maadix in Spain](https://maadix.net/en/terms-conditions).
```

> "Personal data will be treated with the appropriate degree of protection, in accordance with current legislation, with the necessary security measures being taken to avoid its alteration, loss and unauthorised treatment or access by third parties. In no case is there any policy of transferring personal data to third parties. Maadix will not be able to access the data that the client stores in its server, unless the client requests an explicit intervention and provides an access route."

> The Foundation strongly supports individuals’ right to privacy. [Please review our GDPR and Privacy Policy for details.](https://about.publiccode.net/organization/privacy.html)

Reviewer: ```suggestion
The Foundation strongly supports individuals’ right to privacy. [Please read our GDPR and Privacy Policy for more information](https://about.publiccode.net/organization/privacy.html).
```

> The Foundation strongly supports individuals’ right to privacy. [Please read our GDPR and Privacy Policy for more information](https://about.publiccode.net/organization/privacy.html).

Reviewer: ```suggestion
The Foundation strongly supports individuals’ right to privacy. [Read our GDPR and Privacy Policy for more information](../../organization/privacy.md).
```
